web/models/friend.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Entice.Web.Friend do
+  use Ecto.Model
+
+  #has_one :friend_account_id because the 
+  #friend account can be deleted before the 
+  #friend object itself
+  schema "friends" do
+    belongs_to  :account_id,            Entice.Web.Account
```

Comment:
Should be `:account` b/c ecto puts the `_id` on its own, see:
http://hexdocs.pm/ecto/Ecto.Schema.html#belongs_to/3
Section: Options `:foreign_key`